pt/css/README.md

```diff
@@ -285,15 +285,6 @@ Salve esses arquivos e atualize seu site.
 
 ![Figure 14.4](images/final.png)
 
-Look at the code we just pasted to find the places where we added classes in the HTML and used them in the CSS. Where would you make the change if you wanted the date to be turquoise?
```

Comment:
I understand there is a part that wasn't translated yet, but maybe we should translate it instead of just removing it...

Reply:
This part is already translated in the tutorial! That's why I removed it.